Scripts/DCS-BIOS/lib/modules/aircraft_modules/M-2000C.lua

```diff
@@ -0,0 +1,896 @@
+module("M-2000C", package.seeall)
+
+local Functions = require("Functions")
+
+local Module = require("Module")
+
+--- @class M_2000C: Module
+local M_2000C = Module:new("M-2000C", 0x7200, { "M-2000C" })
+
+--v1.39 by Ergo,Matchstick,MisterKnife,WarLord,Espresso29470
+
+--remove Arg# Pilot 1000
+
+local function round(num, numDecimalPlaces, returnstring)
+	if returnstring == nil then
+		returnstring = false
+	end
+	if returnstring then
+		return string.format("%." .. (numDecimalPlaces or 0) .. "f", num)
+	else
+		return tonumber(string.format("%." .. (numDecimalPlaces or 0) .. "f", num))
+	end
+end
```

Comment:
the usages of this function always have `numDecimalPlaces` = `1` and `returnstring` = `true` so I think we can simplify this to just use `tostring(Module.round(num))`

Reply:
fixed